pkg/api/timestamp.go

```diff
@@ -62,15 +82,32 @@ func TimestampResponseHandler(params timestamp.GetTimestampResponseParams) middl
 	}
 
 	// Create response
-	ctx := params.HTTPRequest.Context()
+	httpReq := params.HTTPRequest
+	ctx := httpReq.Context()
 	resp, err := RequestFromRekor(ctx, *req)
 	if err != nil {
 		return handleRekorAPIError(params, http.StatusInternalServerError, err, failedToGenerateTimestampResponse)
 	}
 
-	// TODO: Upload to transparency log and add entry UUID to location header.
-	log.Logger.Errorf("generated OK")
-	return timestamp.NewGetTimestampResponseOK().WithPayload(ioutil.NopCloser(bytes.NewReader(resp)))
+	// Upload to transparency log and add entry UUID to location header.
+	entryParams := entries.CreateLogEntryParams{
+		HTTPRequest:   httpReq,
+		ProposedEntry: createRFC3161FromBytes(resp),
+	}
+
+	logEntry, code, msg, err := createLogEntry(ctx, entryParams)
```

Comment:
I vaguely remember something about this being behind a flag like `--log` or something, not sure if we care about that anymore.

Reply:
Yeah, I think looking back on it we probably want all timestamps to be logged (even if only using as a timestamping machine) just for sake of transparency. But I'm open to creating a default upload flag and making it configurable per req.